.github/workflows/comment-ops.yaml

```diff
@@ -57,16 +57,27 @@ jobs:
       # But it's an overkill for now
       # Instead, we wait a little, and grab the last job that was triggered by a `workflow_dispatch` event, AND is not completed, and we hope for the best 🤞
       - name: Get run URL
+        uses: actions/github-script@v6
         id: run_outputs
-        env:
-          GH_TOKEN: ${{ github.token }}
-        run: |
-          sleep 5
-          url=$(gh run list --repo '${{ github.repository }}' --workflow build.yaml -b ${{ steps.comment-branch.outputs.head_ref }} --json event,url,status -q '[.[] | select(.event=="workflow_dispatch") | select(.status!="completed").url][0]')
-          {
-            echo "run_url=${url}"
-            echo "run_id=${url##*/}"
-          } >> $GITHUB_OUTPUT
+        with:
+          github-token: ${{ github.token }}
+          script: |
+            await new Promise(r => setTimeout(r, 5000));
```

Comment:
I'd love a comment on why this is needed ☺️

Reply:
It's in the comment above :)

```
      # Getting the ID of the workflow we triggered above is a bit tricky, as it's async :(
      # There is a trick we can do - add an input with a unique id, get the list of worklow runs, and check for that input in them until we can find it
      # But it's an overkill for now
      # Instead, we wait a little, and grab the last job that was triggered by a `workflow_dispatch` event, AND is not completed, and we hope for the best 🤞
```